test/test.js

```diff
@@ -207,24 +207,36 @@ describe('Filter', function() {
 
       expect(existsSync(fileForRemoval)).to.be.false;
       expect(existsSync(results.directory + '/a/README.md')).to.be.true;
+      expect(existsSync(results.subject.cachePath + '/a/README.md'), 'OUTPUT: a/README.md should NO LONGER be present').to.be.true;
 
       return results.builder();
     }).then(function(results) {
-      expect(existsSync(results.directory + '/a/README.md'), 'OUTPUT: a/foo.js should NO LONGER be present').to.be.false;
 
-      expect(existsSync(fileForRemoval)).to.be.false;
+     //  expect(existsSync(results.directory + '/a/README.md'), 'OUTPUT: a/README.md should NO LONGER be present').to.be.false;
+       expect(existsSync(results.subject.cachePath + '/a/README.md'), 'OUTPUT: a/README.md should NO LONGER be present').to.be.false;
```

Comment:
yes i think you're right; this has fixed a bug in the test.

👍 

minor thing, but we should still delete the commented line above